yarn.lock

```diff
@@ -28,5 +28,5 @@ select2@^4.0.5:
   integrity sha512-1JeB87s6oN/TDxQQYCvS5EFoQyvV6eYMZZ0AeA4tdFDYWN3BAGZ8npr17UBFddU0lgAt3H0yjX3X6/ekOj1yjw==
 
 themestrap@WISVCH/bootstrap-theme#gh-pages:
-  version "0.3.1"
```

Comment:
So version 0.3.1 has Bootstrap 4 and version 0.3.0 has Bootstrap 3? That's some poor version numbering!

Reply:
yes.